remo/api.py

```diff
@@ -366,18 +366,31 @@ def get_annotation_set(self, id):
         return self.get(url).json()
 
     def export_annotations(
-        self, annotation_set_id: int, annotation_format='json', export_coordinates='pixel', full_path=True, export_tags: bool = True, filter_by_tags: list = None
+        self, 
+        annotation_set_id: int, 
+        annotation_format='json', 
+        export_coordinates='pixel', 
+        full_path=True, 
+        export_tags: bool = True, 
+        filter_by_tags: list = None
     ) -> bytes:
         """
-        Exports annotations in given format
+        Exports annotations in a given file format. 
+        
+        It offers some convenient export options, including:
+        - Methods to append the full_path to image filenames, 
```

Comment:
```suggestion
        It offers some convenient export options, including:
        
        - Methods to append the full_path to image filenames, 
```
We need a space before a bullet list for it to render correctly on the docs